src/Meziantou.Framework.Win32.CredentialManager/CredentialManager.cs

```diff
@@ -14,9 +14,9 @@ namespace Meziantou.Framework.Win32;
 [SupportedOSPlatform("windows5.1.2600")]
 public static class CredentialManager
 {
-    public static unsafe Credential? ReadCredential(string applicationName)
+    public static unsafe Credential? ReadCredential(string applicationName, CredentialType type = CredentialType.Generic)
```

Comment:
Adding an optional parameter is a binary breaking change. You should preserve the original method and add a new overload. (same comment for other new methods)

Reply:
Yes of course! Sorry about that.
I've restored the original method signatures with additional overloads.

Tested by compiling the unit test project from the main branch then replaced the `Meziantou.Framework.Win32.CredentialManager.dll` manually from ones compiled with the latest commit.